server/jobs/onboarding/slack.js

```diff
@@ -0,0 +1,32 @@
+import {removeFromChannel,createPartnerChannels,addToChannel  } from '../../modules/slack/slackIntegration';
+import response from '../../helpers/response';
+
+/**
+ * @function onboardingSlackAutomation
+ * @desc combines all onboarding functions
+ *       into one function.
+ *
+ * @param {string} fellowEmail - user's email
+ * @param {string} removalChannelId - the id of the channel you are removing the developer
+ * @param {string} partnerId - the partner id
+ *
+ * @returns {Object} - With a success message after slack automation has completed
+ */
+export const onboardingSlackAutomation = async (fellowEmail, removalChannelId, partnerId) => {
```

Comment:
The job you created here `onboardingSlackAutomation` is not following our signature of what a job should look like. I would provide some explanation on the Job signature, please refactor afterward. 

Job functions are to receive two arguments, the first is a `placement` object containing the placement data from allocations and the second is a `result` object that the Job can write its result to. A sample would be:
```
const sampleJob = (placement, result) => {
	// Do stuff with the placement info
	// Write the result of the job to the result object
}

export default sampleJob
```
Consequently, a Job is not supposed to return anything or throw any error.

In order to write to the result object, a Job has to create its own key in the result object, the naming of the key should correspond to the Job purpose, for example, in your case, it can be something like `result.slackOnboarding`.

Also, Jobs must be the default export in the file they exist.